### ISSUE TITLE
Snippets working?

### ISSUE THREAD

**Issue by procarrera** [Rocketseat/rocketseat-vscode-reactjs-snippets#23]
Hi guys!
Can you please confirm which snippets are working?
For some reason my VS Code only recognize `rfc`

![Screenshot 2020-10-30 00:34:12](https://user-images.githubusercontent.com/53456120/97657107-cd0b4480-1a47-11eb-8ec8-44e242635522.png)

**Comment by nivynavyblue** [comment]
> Hi guys!
> Can you please confirm which snippets are working?
> For some reason my VS Code only recognize `rfc`
> 
> ![Screenshot 2020-10-30 00:34:12](https://user-images.githubusercontent.com/53456120/97657107-cd0b4480-1a47-11eb-8ec8-44e242635522.png)

Hi, the snippets are working, the problem you're having happens because, besides the rfc  snippets, all the other snippets haven't been created yet.

**Comment by procarrera** [comment]
Oh, ok!!
There is any way to contribute?

Thankss

**Comment by nivynavyblue** [comment]
> Oh, ok!!
> There is any way to contribute?
> 
> Thankss

Yes, you can pick any snippet that wasn't created yet and submit a PR to the owner of the repo.
There's more instruction on contributing inside the README.
